pages/views/sequences.py

```diff
@@ -71,16 +71,20 @@ def get_context_data(self, **kwargs):
 class SequencePageElementView(EnumeratedProgressMixin, TemplateView):
 
     template_name = 'pages/app/sequences/pageelement.html'
-
+    # is_live_event and is_certificate flags not being added
```

Comment:
What does this comment mean, an issue to be fixed or an implementation note to be aware of?